feat: support setting remote images at build-time with `NUXT_IMAGE_DOMAINS` (#1332)
nuxt/image: docs/content/1.get-started/3.providers.md

For more information, you can learn more about the [public directory](https://nuxt.com/docs/guide/directory-structure/public).

::callout

> Image stored in the `assets/` directory are **not** processed with Nuxt Image because those images are managed by Webpack.

Reviewer: ```suggestion
Image stored in the `assets/` directory are **not** processed with Nuxt Image because those images are managed by your bundler (such as Vite or webpack).
```

::

### Remote Images

Using default provider, you can also optimize external URLs. For this, you need to add them to [`domains`](/get-started/configuration#domains) option.

> You can also set the domain using the `NUXT_IMAGE_DOMAINS` environment variable by passing a comma-separated list of domains.

Reviewer: ```suggestion

You can also set the domain using the `NUXT_IMAGE_DOMAINS` environment variable by passing a comma-separated list of domains.
```

For more information, you can learn more about the [public directory](https://nuxt.com/docs/guide/directory-structure/public).

::callout
Image stored in the `assets/` directory are **not** processed with Nuxt Image because those images are managed by your bundler (such as Vite or webpack).
::

### Remote Images

Using default provider, you can also optimize external URLs. For this, you need to add them to [`domains`](/get-started/configuration#domains) option.

> You can also set the domain using the `NUXT_IMAGE_DOMAINS` environment variable by passing a comma-separated list of domains.

Reviewer: ```suggestion
You can also add domains for remote images by setting the `NUXT_IMAGE_DOMAINS` environment variable to a comma-separated list of domains.
```
